src/Countries/US/Ohio/OhioIncome/V20190101/OhioIncome.php

```diff
@@ -32,6 +32,7 @@ public function compute(Collection $tax_areas)
         }
 
         $this->tax_total = $this->payroll->withholdTax($this->getTaxAmountFromTaxBrackets(($this->getAdjustedEarnings() * $this->payroll->pay_periods) - $this->getDependentAllowance(), $this->getTaxBrackets()) / $this->payroll->pay_periods) * 1.075;
+        $this->tax_total += $this->tax_information->additional_withholding;
 
         return round(intval($this->tax_total * 100) / 100, 2);
```

Comment:
not exactly this pr but this should be calling withhold shouldn't it?

Reply:
Yup, updated